sql/core/src/test/scala/org/apache/spark/sql/ReplaceNullWithFalseInPredicateEndToEndSuite.scala

```diff
@@ -19,16 +19,25 @@ package org.apache.spark.sql
 
 import org.apache.spark.sql.catalyst.expressions.{CaseWhen, If, Literal}
 import org.apache.spark.sql.execution.LocalTableScanExec
+import org.apache.spark.sql.execution.adaptive.{
+  AdaptiveSparkPlanHelper,
+  DisableAdaptiveExecutionSuite,
+  EnableAdaptiveExecutionSuite
+}
```

Comment:
Hi, @kazuyukitanimura . This is not Apache Spark style. Please match with the code around your patch.

Reply:
I'm wondering if this came from IDE setting.

Reply:
Updated